conformance/packages/dns-test/src/docker/pdns.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM debian:bookworm-slim
+
+# Install PowerDNS Recursor and Authoritative Server from official repo:
+# https://repo.powerdns.com/
```

Comment:
Unfortunately the `pdns-recursor` package in Debian bookworm is too old to use conveniently. Notably it's missing the `trustanchorfile` configuration element we want to use. Using the 1st party `apt` repository seemed better than building it from source.